Dockerfile.xdmod.init

```diff
@@ -0,0 +1,46 @@
+FROM moc-xdmod-base
```

Comment:
This seems to be largely identical to `Dockerfile.xdmod`. What's the difference between the resulting two images?

Reply:
I think a single image would be best (and then just override the command as necessary in your deployment manifests).

Reply:
I am now creating a single docker file.  In some regards it should be teased apart into 3 different docker files:

1.  to copy the configuration files and source code to their respective directories
2. to run the xdmod-setup script (doesn't really need the web server, as an example)
3. to run xdmid's UI

but that is for a different day

Reply:
I eliminated the Dockerfile.xdmod.init.